shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/session/ConnectionContext.java

```diff
@@ -62,4 +65,11 @@ public void clearTrafficInstance() {
     public void clearCursorConnectionContext() {
         cursorConnectionContext.close();
     }
+    
+    /**
+     * Clear transaction connection context.
+     */
+    public void clearTransactionConnectionContext() {
```

Comment:
Please add clearTransactionConnectionContext in close method.

Reply:
Done. Thank you.